csi/shell/check_bind_mounts.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+VolumeName="$1"
+
+MountPattern="/var/lib/kubelet/pods/.*/volumes/kubernetes\.io~csi/$VolumeName/mount"
```

Comment:
What if kubelet root dir is not "/var/lib/kubelet"? "/var/lib/kubelet" is kubelet root-dir default value. 

refer to kubelet doc: https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/

Reply:
Sorry, I haven't saw this: https://github.com/fluid-cloudnative/fluid/blob/master/charts/fluid/v0.6.0/templates/csi/daemonset.yaml#L49

Maybe kubelet root-dir in mountPath should be set "/var/lib/kubelet" instead of "{{ .Values.csi.kubelet.rootDir }}"?